ui/lib/sync/addon/components/secrets/page/overview.ts

```diff
@@ -52,6 +57,24 @@ export default class SyncSecretsDestinationsPageComponent extends Component<Args
     return this.args.activatedFeatures.includes('secrets-sync');
   }
 
+  get adapterErrorMessage() {
+    // make clear if this is a permissions issue
+    if (this.args.adapterError) {
+      return this.args.adapterError.httpStatus === 403
+        ? 'You do not have read access to the sys/activation-flags endpoint. We cannot determine if Secrets Sync has been activated or not.'
+        : this.args.adapterError.message;
+    }
+    return '';
+  }
+
+  get postErrorMessage() {
+    // make clear if this is a permissions issue
+    if (this.error) {
+      return 'Activation failed. You do not seem to have the permission to enable the feature.';
```

Comment:
"seem to" feels casual for the brand. Suggestion below 👇 
```suggestion
      return 'Activation failed. You may not have the permission to enable the feature.';
```

Reply:
I think we can just return the error from the backend here instead of intercepting ?